src/getCoinCombination.test.js

```diff
@@ -3,8 +3,21 @@
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
-  it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+  it('should return correct combinations for simple cases', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+    expect(getCoinCombination(25)).toEqual([0, 0, 0, 1]);
+  });
+
+  it('should return correct combinations for mixed cases', () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+    expect(getCoinCombination(17)).toEqual([2, 1, 1, 0]);
+  });
+
+  it('should handle large values', () => {
+    expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
+    expect(getCoinCombination(123)).toEqual([3, 0, 2, 4]);
```

Comment:
The expected output for `getCoinCombination(123)` seems incorrect. Ensure the result reflects the smallest number of coins needed to make up the amount. Double-check the logic for calculating the coin combination.